pulsar-broker-common/src/main/java/org/apache/pulsar/bookie/rackawareness/IsolatedBookieEnsemblePlacementPolicy.java

```diff
@@ -59,6 +58,8 @@
 
     private MetadataCache<BookiesRackConfiguration> bookieMappingCache;
 
+    private static final String PULSAR_METADATA_ISOLATION_GROUP = "*";
```

Comment:
This name is a little confusing, any better choice?

Reply:
changed to PULSAR_SYSTEM_TOPIC_ISOLATION_GROUP